dynamic-layers/qt5-layer/recipes-qt/qt5/qtbase_%.bbappend

```diff
@@ -16,16 +16,33 @@ SRC_URI_append_imxgpu3d = " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'x11', '', '${SRC_URI_APPEND_3D_NOT_X11}', d)} \
 "
 
+PACKAGECONFIG_GBM_IMXGPU_MX8_mx8 = "gbm kms"
+PACKAGECONFIG_GBM_IMXGPU_imxgpu  = "${PACKAGECONFIG_GBM_IMXGPU_MX8}"
```

Comment:
```suggestion
PACKAGECONFIG_IMX_GPU = ""
PACKAGECONFIG_IMX_GPU_mx8 = "gbm kms"
```

Reply:
Okay, but I think we should at least include the GL for context? `PACKAGECONFIG_GL_IMX_GPU`?